tests/Fixtures/TestCustomEndpointMetadata.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace Knuckles\Scribe\Tests\Fixtures;
+
+use Knuckles\Camel\Extraction\ExtractedEndpointData;
+use Knuckles\Scribe\Extracting\Strategies\Strategy;
+
+class TestCustomEndpointMetadata extends Strategy
+{
+    public function __invoke(ExtractedEndpointData $endpointData, array $routeRules): ?array
+    {
+        $endpointData->metadata->custom['myProperty'] = 'some custom metadata';
+        return null;
```

Comment:
I don't think this matters, does it? Functions in PHP implicitly return null.

Oh, this is because of the type annotation requiring a return statement.